test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -15,3 +15,21 @@ Describe "Get-Process" -Tags "CI" {
         Get-Process | foreach-object { $_.Name | Should Not BeNullOrEmpty }
     }
 }
+
+Describe "Get-Process Formatting" -Tags "Feature" {
+    It "Should not have Handle in table format header" {
+        $types = "System.Diagnostics.Process","System.Diagnostics.Process#IncludeUserName"
+
+        foreach ($type in $types)
+        {
+            $formatData = Get-FormatData -TypeName $type
```

Comment:
By default, `Get-FormatData` doesn't return all possible formats because previous versions of PowerShell use that cmdlet to get the formatting data and versions prior to 5.1 couldn't handle all the varieties of possible format data.

To get all the format data, you need to specify an additional parameter, like this:

```powershell
$formatData = Get-FormatData -TypeName $type -PowerShellVersion $PSVersionTable.PSVersion
```